app/controllers/CalculationListController.scala

```diff
@@ -31,6 +33,8 @@ import scala.concurrent.{ExecutionContext, Future}
 
 class CalculationListController @Inject()(val authentication: AuthenticationPredicate,
                                           val calculationListService: CalculationListService,
+                                          val hipCalculationListConnector: CalculationListLegacyConnector,
+                                          val appConfig: MicroserviceAppConfig,
                                           cc: ControllerComponents)(implicit ec: ExecutionContext) extends BackendController(cc) with Logging {
   // 1404
```

Comment:
can we add a little more context to this comment or bake the information in method naming or code?